build/content.js

```diff
@@ -450,6 +475,40 @@ function sendMessage(command, options) {
     });
 }
 
+const userHistoryKey = 'user-history';
+function getHistory() {
+    return new Promise(resolve => {
+        chrome.storage.local.get([userHistoryKey], function (result = []) {
+            resolve(result[userHistoryKey] || []);
+        });
+    });
+}
+function pushHistory(commandKey) {
+    return __awaiter(this, void 0, void 0, function* () {
+        const hist = yield getHistory();
+        const newHist = [...new Set([commandKey, ...hist])];
+        return new Promise(resolve => {
+            chrome.storage.local.set({ [userHistoryKey]: newHist }, () => {
+                resolve(newHist);
+            });
+        });
+    });
+}
+
+let commands$1 = commands;
```

Comment:
What's the $ for?

Reply:
I don't know. This is the built file so we don't manage it

Reply:
🤦  built file